### PR TITLE
Payments/integration

### PR DESCRIPTION
**Pull request by eliasmalik**
Integrate the backend payments API with the React checkout form component.

Related to #2

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -30,7 +30,8 @@
     "style-loader": "^0.13.0",
     "tape": "^4.5.1",
     "webpack": "^1.12.14",
-    "webpack-dev-server": "^1.14.1"
+    "webpack-dev-server": "^1.14.1",
+    "babel-plugin-transform-object-rest-spread": "^6.6.5"
```
Comment (franzmoro) [inline frontend/package.json]: we have the object-rest-spread as both a dev-dependency and as a dependency.... reduce this to either one or the other?

File: frontend/src/js/Checkout/index.jsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      </div>
+    )
+  }
+}
+
+const clickHandler = (e) => {
+  e.preventDefault()
+  const paymentForm = document.getElementById('payment-form')
+  const stripeResponseHandler = (status, response) => {
+    // TODO: delete next 2 lines when we join up with the basket
```
Comment (franzmoro) [inline frontend/src/js/Checkout/index.jsx:61]: reminder!!! @eliascodes @tu6619  do you still need to do this??

File: frontend/src/js/footer/bottomFooter.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import Logo from './logo.js'
+import Copyright from './copyright.js'
+
+class BottomFooter extends React.Component {
+  render () {
+    return (
+      <div>
+        <Logo />
+        <Copyright copyright='© 1996-2016, Amazon 2.0, Inc. or its affiliates'/>
```
Comment (franzmoro) [inline frontend/src/js/footer/bottomFooter.js:10]: :joy:

Comment (katbow) [inline frontend/src/js/footer/bottomFooter.js:10]: Copyright haha

**Pull request merged**